docs/public/requirements.txt

```diff
@@ -0,0 +1,2 @@
+breathe
```

Comment:
Do we need to add `sphinx` too in install dependencies, or the plugins installation will take care of it automatically ?